pg_backup_api/pg_backup_api/logic/utility_controller.py

```diff
@@ -48,7 +48,7 @@ def diagnose(self):
         stored_output = json.loads(output._writer.json_output["_INFO"][-1])
 
         diag_output = DiagnoseOutput(
-            _global=str(stored_output["global"]), servers=str(stored_output["servers"])
+            _global=str(stored_output["global"]), servers=json.dumps(stored_output["servers"])
```

Comment:
I think we also need a `json.dumps` for `stored_output["global"]`

Reply:
Good catch, thank you